### ISSUE TITLE
Add default certificate option

### ISSUE THREAD

**Issue by clementblaise** [pomerium/ingress-controller#144]
**Is your feature request related to a problem? Please describe.**

Having to rely on a certificate per Ingress

**Describe the solution you'd like**

Allow the usage of a default certificate, similar to nginx with the following option

```yaml
extraArgs: { default-ssl-certificate: "<namespace>/<secret>" }
```

**Explain any additional use-cases**

I believe it is much more simple to manage a single wildcard for the Ingress Class. Issuing certificate per ingress means more certificate that can expire and the potential to hit rate limiting on let's encrypt

**Comment by wasaga** [comment]
currently you may just create a wildcard `Certificate` using `cert-manager` to account for automatic renewal, and refer to it in the TLS section of the `Ingress`. Multiple Ingresses may refer to a single TLS secret, and we do support wildcard certificates.

**Comment by clementblaise** [comment]
Hello @wasaga I agree that's what I did first but the limitation is that it won't work for Ingress outside the namespace, since you cannot load a secret from another namespace. Provisioning a wildcard in each of the namespace that will be using it kind of defeat the purpose.

**Comment by travisgroth** [comment]
Closed by #151
